src/test/java/com/epam/tc/hw2/ex1/ExerciseOneTest.java

```diff
@@ -0,0 +1,161 @@
+            driver.findElement(By.id("login-button")).click();
+        } catch (NoSuchElementException e) {
+            System.err.println("No element: " + e);
+        }
+
+        List<WebElement> searchResults = driver.findElements(By.id("user-name"));
+
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(searchResults.size() > 0).as("Login failed").isTrue();
+        softly.assertThat(driver.findElement(By.id("user-name")).getText()).isEqualTo("ROMAN IOVLEV");
+        softly.assertAll();
+
+    }
+
+    //Assert that there are 4 items on the header section are displayed and they have proper texts
+    @Test
+    public void navBarCheck() {
+
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a[href='index.html']")).isDisplayed());
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a[href='contacts.html']")).isDisplayed());
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a.dropdown-toggle")).isDisplayed());
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a[href='metals-colors.html']")).isDisplayed());
+
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a[href='index.html']")).getText()).isEqualTo("HOME");
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a[href='contacts.html']")).getText()).isEqualTo("CONTACT FORM");
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a.dropdown-toggle"))
+                .getText()).isEqualTo("SERVICE");
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a[href='metals-colors.html']")).getText()).isEqualTo("METALS & COLORS");
+        softly.assertAll();
+
+    }
+
+    //Assert that there are 4 images on the Index Page and they are displayed
+    @Test
+    public void imagesCheck() {
+
+        SoftAssertions softly = new SoftAssertions();
+        List<WebElement> searchResults = driver.findElements(By.className("benefit-icon"));
+        softly.assertThat(searchResults.size() == 4).as("Images quantity inconsistency").isTrue();
+
+        softly.assertThat(driver.findElement(By.className("icon-practise")).isDisplayed());
+        softly.assertThat(driver.findElement(By.className("icon-custom")).isDisplayed());
+        softly.assertThat(driver.findElement(By.className("icon-multi")).isDisplayed());
+        softly.assertThat(driver.findElement(By.className("icon-base")).isDisplayed());
+
+        softly.assertAll();
+
+    }
+
+    //Assert that there are 4 texts on the Index Page under icons and they have proper text
+    @Test
+    public void textCheck() {
+
+        SoftAssertions softly = new SoftAssertions();
+        List<WebElement> searchResults = driver.findElements(By.className("benefit-txt"));
+        softly.assertThat(searchResults.size() == 4).as("Texts quantity inconsistency").isTrue();
+
+        softly.assertThat(driver
```

Comment:
You don't need the whole text here :)

Reply:
check is modified, unnecessary selectors are removed, check is now performed via "for" loop